src/mapper.rs

```diff
@@ -52,3 +52,13 @@ pub trait Mapper {
     /// ```
     fn unmap(&mut self, virt_start: usize, bytes: usize);
 }
+
+/// an identity mapper which maps a physical address into itself.
```

Comment:
(Please wrap lines into 80 columns if you apply this change.)
```suggestion
/// An identity mapper which maps a physical address into the virtual address of the same value.
///
/// # Safety
///
/// Users of this mapper must ensure that the identity mapping is correctly set up, otherwise memory safety will be violated.
```